include/envoy/protobuf/message_validator.cc

```diff
@@ -0,0 +1,24 @@
+#include "envoy/protobuf/message_validator.h"
+
+#include "envoy/common/exception.h"
+
+#include "common/common/logger.h"
+
+#include "absl/strings/str_cat.h"
+
+namespace Envoy {
+namespace ProtobufMessage {
+
+void ValidationVisitor::onDeprecatedFieldDefault(absl::string_view description,
```

Comment:
We tend to prefer that even default behavior lives in `source/` vs. `include/`, the latter of which is generally just pure virtual functions and classes.